builder/bundleset.go

```diff
@@ -12,13 +13,23 @@ import (
 )
 
 var (
-	validBundleNameRegex  = regexp.MustCompile(`^[A-Za-z0-9-_]+$`)
-	validPackageNameRegex = regexp.MustCompile(`^[A-Za-z0-9-_+.]+$`)
+	validBundleNameRegex   = regexp.MustCompile(`^[A-Za-z0-9-_]+$`)
+	validPackageNameRegex  = regexp.MustCompile(`^[A-Za-z0-9-_+.]+$`)
+	bundleHeaderFieldRegex = regexp.MustCompile(`^# \[([A-Z]+)\]:\s*(.*)$`)
 )
 
+type bundleHeader struct {
```

Comment:
Optional: my first take with this would be just create a `map[string]string` (since we don't really care much about the actual strings at the moment to give them attributes).

Reply:
Input taken, but I chose to leave it this way, for a couple reasons:

1. For now we have a fixed, consistent header, but it could change. I think it's good to have the programatic definition of that strictly defined in one place, as opposed to implicitly defined by what map keys get looked up in different places in the code. 
1. This makes it easier to write code that uses it. `b.Header.Fake` is a compile-time error, while `b.Header['fake']` is a runtime error.
1. At some point we'll likely switch to using TOML (or JSON) for bundle definition files. Parsing libraries for those usually require a type defined for it to map to. While the format might not end up exactly like what I have here (it might not be in a nested `Header` object), this gets us a head start.